### PR TITLE
Send key vault audit logs to log analytics workspace

### PR DESCRIPTION
**Pull request by hannah38**
when new cftapps key vault is created

Locally tested for sandbox and develop

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -14,6 +14,9 @@ case "${1}" in
 		ENV="sbox"
     LONG_ENV="sandbox"
     CRITICALITY="Low"
+    LOG_NAME="hmcts-sbox-law"
```
Comment (timja) [inline run.sh]: this should go to `hmcts-sandbox`

```suggestion
    LOG_NAME="hmcts-sandbox"
```

File: run.sh [hunk 2]
```diff
@@ -14,6 +14,9 @@ case "${1}" in
...
+    LOG_RG="oms-automation-rg"
+    LOG_SUB="b72ab7b7-723f-4b18-b6f6-03b0f2c6a1bb"
```
Comment (timja) [inline run.sh]:
```suggestion
    LOG_SUB="bf308a5c-0624-4334-8ff8-8dca9fd43783"
```

File: run.sh [hunk 3]
```diff
@@ -14,6 +14,9 @@ case "${1}" in
 		ENV="sbox"
     LONG_ENV="sandbox"
     CRITICALITY="Low"
+    LOG_NAME="hmcts-sbox-law"
+    LOG_RG="oms-automation-rg"
```
Comment (timja) [inline run.sh]:
```suggestion
    LOG_RG="oms-automation"
```

Comment (hannah38) [reply run.sh]: Corrected Sandbox Log Analytics Workspace

File: run.sh [hunk 4]
```diff
@@ -164,7 +172,18 @@ az keyvault create --name ${VAULT_NAME} \
   --enabled-for-deployment true  \
   --tags "${COMMON_TAGS[@]}" \
   --enabled-for-template-deployment true
-  
+
+if [ ${SUB} == "cftapps" ]; then
```
Comment (timja) [inline run.sh]: I don't think this if is needed you're missing cftsbox and cftptl with this

Comment (hannah38) [reply run.sh]: The if statement has been removed

**Review by timja — APPROVED** [review]

**Pull request merged**